comictaggerlib/ui/taggerwindow.ui

```diff
@@ -981,6 +981,44 @@
                   </property>
                  </widget>
                 </item>
+                <item row="4" column="0">
+                 <widget class="QLabel" name="lblCommunityRating">
+                  <property name="text">
+                   <string>Community Rating</string>
+                  </property>
+                 </widget>
+                </item>
+                <item row="4" column="1">
+                 <widget class="QDoubleSpinBox" name="dsbCommunityRating">
+                  <property name="sizePolicy">
+                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+                    <horstretch>0</horstretch>
+                    <verstretch>0</verstretch>
+                   </sizepolicy>
+                  </property>
+                  <property name="maximumSize">
+                   <size>
+                    <width>80</width>
+                    <height>16777215</height>
+                   </size>
+                  </property>
+                  <property name="acceptDrops">
+                   <bool>false</bool>
+                  </property>
+                  <property name="decimals">
+                   <number>1</number>
+                  </property>
+                  <property name="minimum">
+                   <double>0.000000000000000</double>
+                  </property>
+                  <property name="maximum">
+                   <double>5.000000000000000</double>
+                  </property>
+                  <property name="singleStep">
+                   <double>0.100000000000000</double>
+                  </property>
+                 </widget>
+                </item>
```

Comment:
I would recommend unsetting sizePolicy and acceptDrops as they don't provide any use over the default.
I would also recommend changing maximumSize to minimumSize with a width of 80 so that it will match the criticalRating box above it.

```diff
diff --git a/comictaggerlib/ui/taggerwindow.ui b/comictaggerlib/ui/taggerwindow.ui
index 0d67f8c..90d5c31 100644
--- a/comictaggerlib/ui/taggerwindow.ui
+++ b/comictaggerlib/ui/taggerwindow.ui
@@ -990,21 +990,12 @@
                 </item>
                 <item row="4" column="1">
                  <widget class="QDoubleSpinBox" name="dsbCommunityRating">
-                  <property name="sizePolicy">
-                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
-                    <horstretch>0</horstretch>
-                    <verstretch>0</verstretch>
-                   </sizepolicy>
-                  </property>
-                  <property name="maximumSize">
+                  <property name="minimumSize">
                    <size>
                     <width>80</width>
-                    <height>16777215</height>
+                    <height>0</height>
                    </size>
                   </property>
-                  <property name="acceptDrops">
-                   <bool>false</bool>
-                  </property>
                   <property name="decimals">
                    <number>1</number>
                   </property>
```

Reply:
Hmm... must be a difference between the version of Qt Designer that I have, but it indicated that the properties were set manually I just unset them in Qt Designer